.gitignore

```diff
@@ -18,3 +18,6 @@ erl_crash.dump
 
 # Also ignore archive artifacts (built via "mix archive.build").
 *.ez
+
+#asdf
```

Comment:
.tool-version is used not only by asdf so I think this comment can be removed

Reply:
okay